indico/web/client/js/react/components/MathJax.jsx

```diff
@@ -0,0 +1,23 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React, {useEffect, useRef} from 'react';
+import PropTypes from 'prop-types';
+
+export default function MathJax({children}) {
+  const wrapperRef = useRef(null);
+
+  useEffect(() => {
+    jQuery(wrapperRef.current).mathJax();
```

Comment:
`$` instead of `jQuery`?

Reply:
But let's just export the non-jquery version of this and use that one here. Because all the jQuery one does is this:

```js
  $.fn.mathJax = function() {
    $(this).each(function() {
      pd.mathJax(this);
    });
    return this;
  };
```